tests/spatial_analyses/cell_based_association_test.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+
+"""Created on Thu Jun  1 09:28:42 2023.
+
+@author: vella.
+"""
+
+from pathlib import Path
+
+import geopandas as gpd
+
+import eis_toolkit.spatial_analyses.cell_based_association as cba
+
+parent_dir = Path(__file__).parent.parent
+
+vector_path = str(parent_dir.joinpath("data/remote/Test_Litho.geojson"))
+points_path = str(parent_dir.joinpath("data/remote/Test_Occ.geojson"))
+lines_path = str(parent_dir.joinpath("data/remote/Test_Faults.geojson"))
+matrix_path = str(parent_dir.joinpath("data/remote/Test_CBA_matrix.geojson"))
+
+vector_file = gpd.GeoDataFrame.from_file(vector_path)
+points_file = gpd.GeoDataFrame.from_file(points_path)
+lines_file = gpd.GeoDataFrame.from_file(lines_path)
+
+
+def test_crs():
+    """Test that Coordinate Systems for input dataset and output grids are equivalent."""
+    grid, cba_grid = cba.init_from_vector_data(
+        cell_size=4000, geodataframe=vector_file, column="Litho", subset_of_target_attribute_values="all"
```

Comment:
If the option of passing `"all"` is removed as suggested in other comment, these all should be changed accordingly:

```suggestion
        cell_size=4000, geodataframe=vector_file, column="Litho", subset_of_target_attribute_values=None
```